packages/node_modules/@ciscospark/widget-space/README.md

```diff
@@ -271,6 +272,39 @@ ciscospark.widget(widgetEl).on('messages:created', function(e) {
 
 All available events are outlined in our [events guide](./events.md).
 
+### Answering Incoming Calls and Meetings <a id="incoming"></a>
+
+In order to answer an incoming call you may do the following depending on what information
+is available when answering the call:
+
+- if it is a space meeting, the `spaceId`
+
+  ```js
+    ciscospark.widget(widgetEl).spaceWidget({
+      accessToken: 'AN_ACCESS_TOKEN',
+      destinationType: 'spaceId',
+      destiationId: 'SPACE_ID'
+  });
+  ```
+
+- a `call` object provided by the `calls:created` event from the [`recents widget`](../widget-recents/events.md#callscreated)
+
+  ```js
+    ciscospark.widget(widgetEl).spaceWidget({
+      accessToken: 'AN_ACCESS_TOKEN',
+      call: callObject
+    });
+  ```
+
+- a `call` object provided by the `calls:created` event from the [`recents widget`](../widget-recents/events.md#callscreated)
```

Comment:
Did you mean to use `calls:incoming` event on this second (duplicate) block?

Reply:
🤦‍♂️